src/components/spaces/cell-zoom/OrganelleWrapper.tsx

```diff
@@ -0,0 +1,388 @@
+          {hoverLabel}
+        </div>)
+      : null;
+
+    const droppers: any = this.state.dropperCoords.map((dropperCoord: any, i: number) => (
+      <div className="temp-dropper" key={i} style={{left: dropperCoord.x - 6, top: dropperCoord.y - 28}}>
+        <img src="assets/cell-zoom/dropper.png" width="32px"/>
+      </div>
+    ));
+    // const dropperCursor = this.state.hoveredOrganelle && this.isModeDropper(rootStore.mode);
+    const dropperCursor = false;
+    const style = {height: MODEL_HEIGHT, width: MODEL_WIDTH};
+    return (
+      <div className={"model-wrapper" + (dropperCursor ? " dropper" : "")} style={style}>
+        <div id={this.props.elementName} className="model" onMouseLeave={this.resetHoveredOrganelle}/>
+        {hoverDiv}
+        {droppers}
+      </div>
+    );
+  }
+
+  private getModel() {
+    // return appStore.boxes.get(this.props.boxId).model;
+    return this.model;
+  }
+
+  private completeLoad() {
+    const model = this.getModel();
+    model.on("view.loaded", () => {
+      this.updateReceptorImage();
+    });
+
+    model.setTimeout(
+      () => {
+        for (let i = 0; i < 3; i++) {
+          // The world could have been unmounted since the timeout was set
+          if (model && model.world) {
+            model.world.createAgent(model.world.species.gProtein);
+          }
+        }
+      },
+      1300);
+
+    model.on("hexagon.notify", () => this.updateReceptorImage());
+
+    model.on("gProtein.notify.break_time", (evt: any) => {
+      const proteinToBreak = evt.agent;
+      const location = {x: proteinToBreak.getProperty("x"), y: proteinToBreak.getProperty("y")};
+      const body = model.world.createAgent(model.world.species.gProteinBody);
+      body.setProperties(location);
+
+      const part = model.world.createAgent(model.world.species.gProteinPart);
+      part.setProperties(location);
+
+      proteinToBreak.die();
+
+      model.world.setProperty("g_protein_bound", false);
+
+      model.world.createAgent(model.world.species.gProtein);
+    });
+
+    model.on("model.step", () => {
+      // const organism: IOrganism = appStore.getBoxOrganism(this.props.boxId);
+      // let percentLightness = organism.lightness;
+      let percentLightness = .2;
+
+      if (percentLightness <= 0.19) {
+        percentLightness = 0;
+      } else if (percentLightness < 0.39) {
+        percentLightness = 0.2;
+      } else if (percentLightness < 0.59) {
+        percentLightness = 0.4;
+      } else if (percentLightness < 0.79) {
+        percentLightness = 0.6;
+      } else if (percentLightness < 0.99) {
+        percentLightness = 0.8;
+      } else {
+        percentLightness = 1;
+      }
+
+      // go from lightest to darkest in HSL space, which provides the best gradual transition
+
+      // lightest brown: rgb(244, 212, 141) : hsl(41°, 82%, 75%)
+      // darkest brown:  rgb(124, 81, 21)   : hsl(35°, 71%, 28%)
+
+      const light = [41, 82, 75];
+      const dark = [35, 71, 28];
```

Comment:
I realize we're deep into migrated code at this point, but a general related question - are the colors for mice going to be globally defined? If so, what's the best color format for consistency and ease-of-use? Start at RGB and have helpers to provide conversion to HSL & hex values?

Reply:
Sounds like a good idea for mouse colors to get a global definition...could be nice to share colors across graphs and cells, for example.